cpp/src/Ice/SSL/SchannelEngine.cpp

```diff
@@ -1012,7 +1268,7 @@ Schannel::SSLEngine::newCredentialsHandle(bool incoming) const
     }
     else
     {
-        cred.dwFlags = SCH_CRED_MANUAL_CRED_VALIDATION | SCH_CRED_NO_SERVERNAME_CHECK | SCH_CRED_NO_DEFAULT_CREDS;
+        cred.dwFlags = SCH_CRED_NO_SERVERNAME_CHECK | SCH_CRED_NO_DEFAULT_CREDS;
```

Comment:
Removed `SCH_CRED_MANUAL_CRED_VALIDATION` we want to use the default validation mechanism when possible. Namely when validating a server certificate and the client didn't configure a custom trust store or validation callback.